src/crypto/public_key/elliptic/ecdh_x.js

```diff
@@ -28,7 +30,12 @@ export async function generate(algo) {
       const k = getRandomBytes(32);
       k[0] &= 248;
       k[31] = (k[31] & 127) | 64;
```

Comment:
```suggestion

```
No need for this anymore, it only applied to LegacyCurve25519, see [spec](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-10.html#name-curve25519legacy-ecdh-secre).